### DISCUSSION TITLE
static files

### DISCUSSION THREAD

**Discussion by Sokhavuth** [ahkohd/express-php#6] [General]
I'm glad to have your response. 

As I am a web development trainer at Computer Academy STEP Cambodia (https://www.facebook.com/itstepcambodia) I love your Express-php web framework that I was looking for since longtime ago. However, I am unable to use static asset in the "public" folder set up in the config.php file. If you could explain me how to solve this problem, it would wonderful.

However, I decided to use your web framework as a serverless PHP framework by hosting static assets on a separated server on Vercel platform. And it works fine for me, - I could use css, js, and image files without any problem. You could take a look on the website using your framework I just deployed yesterday to Vercel.app here: https://tvp.vercel.app/ 

Here is the link to the repository using your web framework: https://github.com/Sokhavuth/tvp

**Comment by ahkohd** [comment]
Hi, @Sokhavuth  thanks for reaching out. 

There  is a example of how to use static assets in your public folder.
```php
<img src="<?php echo $data["app"]->basePath; ?>/public/img/some-picture.jpg" />
```

Please update your Express.php file, I have pushed some commits to fix some bugs. I forgot to push this a year ago 😂.
